tests/iDibo_tests.py

```diff
@@ -1,11 +1,35 @@
-from nose.tools import *
-import iDibo
+import unittest
+from gluon.shell import exec_environment
+from gluon.contrib.test_helpers import form_postvars
```

Comment:
Ok, the problem is gluon is a folder in the root web2py directiory and so
far i've been working with ad pushing files from web2py/application/iDibo,
gluon is in web2py/gluon, so what do i do now?

On Tue, Feb 3, 2015 at 7:44 AM, Oladayo Satope <satopeoladayo@gmail.com>
wrote:

>
> I'll check to confirm I committed the file. I'll appreciate u look into
> the last question I asked on the main project about the db and contoller
> for listing items in campaigns table. Thanks
>
>
>
> Best Regards,
> 'Yemisi
>
> Sent from my Samsung Tablet
>
>
> I'll check to confirm I committed the file. I'll appreciate u look into
> the last question I asked on the main project about the db and contoller
> for listing items in campaigns table. Thanks
>
>
>
> Best Regards,
> 'Yemisi
>
> Sent from my Samsung Tablet
> Ana Oprea <notifications@github.com> wrote:
>
> In tests/iDibo_tests.py
> <https://github.com/anaoprea/iDibo/pull/8#discussion_r23953527>:
>
> > @@ -1,11 +1,35 @@
> > -from nose.tools import *
> > -import iDibo
> > +import unittest
> > +from gluon.shell import exec_environment
> > +from gluon.contrib.test_helpers import form_postvars
>
> Where can I find this module? I don't see it in
> https://github.com/web2py/web2py/tree/master/gluon/contrib
>
> Do you also get the error below?
> $ python web2py.py -S iDibo -M -R applications/iDibo/tests/iDibo_tests.py
> Version 2.9.11-stable+timestamp.2014.09.15.23.35.11
> Traceback (most recent call last):
> File "/home/ana/dev/learnitgirl/web2py/gluon/shell.py", line 262, in run
> execfile(startfile, _env)
> File "applications/iDibo/tests/iDibo_tests.py", line 3, in <module>
> from gluon.contrib.test_helpers import form_postvars
> File "/home/ana/dev/learnitgirl/web2py/gluon/custom_import.py", line 105,
> in custom_importer
> return NATIVE_IMPORTER(name, globals, locals, fromlist, level)
> ImportError: No module named test_helpers
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/anaoprea/iDibo/pull/8/files#r23953527>.
>
>
> In tests/iDibo_tests.py:
>
> > @@ -1,11 +1,35 @@
> > -from nose.tools import *
> > -import iDibo
> > +import unittest
> > +from gluon.shell import exec_environment
> > +from gluon.contrib.test_helpers import form_postvars
> Where can I find this module? I don't see it in
> https://github.com/web2py/web2py/tree/master/gluon/contrib
>
> Do you also get the error below?
> $ python web2py.py -S iDibo -M -R applications/iDibo/tests/iDibo_tests.py
> Version 2.9.11-stable+timestamp.2014.09.15.23.35.11
> Traceback (most recent call last):
> File "/home/ana/dev/learnitgirl/web2py/gluon/shell.py", line 262, in run
> execfile(startfile, _env)
> File "applications/iDibo/tests/iDibo_tests.py", line 3, in <module>
> from gluon.contrib.test_helpers import form_postvars
> File "/home/ana/dev/learnitgirl/web2py/gluon/custom_import.py", line 105,
> in custom_importer
> return NATIVE_IMPORTER(name, globals, locals, fromlist, level)
> ImportError: No module named test_helpers
>
> --
> Reply to this email directly or view it on GitHub.
>
>
>
>


-- 



*We are what we repeatedly do. Excellence, therefore, is not an act but a
habit.!!!! Best Regards, 'Yemisi*
*+2348069385406*